src/ilcdlib/entity/lcia.py

```diff
@@ -57,10 +58,28 @@ def get_impact_set(self, scenario_names: dict[str, str]) -> ImpactSet:
                 scenario_names=scenario_names,
                 scope_to_units_mapper=self.scope_to_units_mapper,
             )
+
+        self._extract_and_set_a1a2a3_impact(impacts)
+
         if len(ext) == 0:
             del impacts["ext"]
         return ImpactSet(**impacts)  # type: ignore
 
+    def _extract_and_set_a1a2a3_impact(self, impacts: dict[str, ScopeSet | dict]):
+        """Set A1A2A3 value if None provided."""
+
+        def _process(scope_set: ScopeSet):
+            if scope_set.A1A2A3 is None and scope_set.A1 and scope_set.A2 and scope_set.A3:
+                s = scope_set.A1.mean + scope_set.A2.mean + scope_set.A3.mean
```

Comment:
I'm wondering if it is possible we have no let's say A1 or A3... Probably what we should do in case A1A2A3 is missing:
1. Check if at least one of A1, A2, A3 exist
2. Sum everything which exist
3. Make sure the units are compatible, if not - throw error